themes/jquery/header.php

```diff
@@ -64,7 +64,7 @@
 					<li><a href="https://plugins.jquery.com/">Plugins</a></li>
 					<li class="dropdown"><a href="https://contribute.jquery.org/">Contribute</a>
 						<ul>
-							<li><a href="https://js.foundation/CLA">CLA</a></li>
+							<li><a href="https://cla.js.foundation/jquery/">CLA</a></li>
```

Comment:
Can we do this in a separate PR? :)

Reply:
of course, I'll do a sep. pr for all the link updates....